addons/common/functions/fnc_isRemoteControlled.sqf

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+/*
+ * Author: NeilZar
+ * Checks if the given unit is currently remote controlled.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Unit is Remote Controlled <BOOL>
+ *
+ * Example:
+ * [_unit] call zen_common_fnc_isRemoteControlled
+ *
+ * Public: Yes
+ */
+
+params ["_unit"];
+
+private _curators = allCurators apply {getAssignedCuratorUnit _x} select {!(isNull _x)};
+
+_unit getVariable ["bis_fnc_moduleremotecontrol_owner", objNull] in _curators
```

Comment:
Can use an `!isNull` check after #521 .

Reply:
With the current `!isNull` select on line 20 there should be no issue, unless I am missing something. There should be no `objNull` in `_curators` to cause a false positive in case the `getVariable` returns `objNull`

Reply:
I meant just: `!isNull (_unit getVariable ["bis_fnc_moduleremotecontrol_owner", objNull])` since #521 makes it so the variable is reset after disconnect.

Either is fine though really. Only suggestion would be to remove the parentheses around `isNull` on L20.